lib/compiler.ts

```diff
@@ -38,7 +38,81 @@ export function getCompilerScriptTarget(target: ScriptTarget) {
   }
 }
 
-export function getCompilerLibOption(target: ScriptTarget): string[] {
+// Created from https://github.com/dsherret/ts-morph/blob/latest/packages/common/src/data/libFiles.ts
+export type LibFileName =
+  | "lib.d.ts"
```

Comment:
It's this way because this is what the compiler API seems to work with and I don't see a convenience function for going from tsconfig lib option to this.

Reply:
Using `ts.libMap` is how you convert it.  It is just an internal, but it is there.

Reply:
Nice. Thanks! I will update this tomorrow

Reply:
Implemented in #167